CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ on patches that are being contributed.
 
 The sign-off is a simple line at the end of the explanation for the
 patch, which certifies that you wrote it or otherwise have the right
-to pass it on as an open-source patch.  The rules are pretty simple:
+to pass it on as an open-source patch. The rules are pretty simple:
```

Comment:
s/open-source/open source

Reply:
`open-source` is valid too: https://en.wikipedia.org/wiki/Open-source_model